messaginginapp/src/main/java/io/customer/messaginginapp/gist/presentation/GistSdk.kt

```diff
@@ -25,12 +25,12 @@ const val GIST_TAG: String = "[CIO]"
 object GistSdk : Application.ActivityLifecycleCallbacks {
     private const val SHARED_PREFERENCES_NAME = "gist-sdk"
     private const val SHARED_PREFERENCES_USER_TOKEN_KEY = "userToken"
-    private const val POLL_INTERVAL = 10_000L
 
     private val sharedPreferences by lazy {
         application.getSharedPreferences(SHARED_PREFERENCES_NAME, MODE_PRIVATE)
     }
 
+    internal var pollInterval = 600_000L
```

Comment:
Why is the default poll value increased?

Reply:
This will be our new default, the server will then adjust it remotely, you can read more here: [INAPP-12248](https://linear.app/customerio/issue/INAPP-12248/update-the-queue-composer-to-automatically-set-polling-interval-based)